packages/eslint-config-nimble-core/package.json

```diff
@@ -30,7 +30,7 @@
     "lib": "lib"
   },
   "engines": {
-    "node": "^14.18.2 || ^16 || ^17 || ^18"
+    "node": "^14.18.2 || >=16"
```

Comment:
Do we still need to support Node 14?

Reply:
We do
![image](https://github.com/nimblehq/eslint-config-nimble/assets/42540264/3141233d-b439-4080-b257-706dbc7d45df)